### ISSUE TITLE
Reproduction results

### ISSUE THREAD

**Issue by chenxiaodanhit** [yongkyung-oh/Stable-Neural-SDEs#3]
Thank you for your great contributions! I have a question regarding classification with irregular samples. I run the [sepsis-sde.py](https://github.com/yongkyung-oh/Stable-Neural-SDEs/blob/main/benchmark_classification/sepsis-sde.py) with model name of Neural LSDE. However, the AUROC I achieved is only 82.92%, which is significantly lower than 90.0% reported in the paper. Could you please provide some details on how to accurately reprodece the results?

**Comment by yongkyung-oh** [comment]
Thank you for your comment. Do you have the same issue for the other models with optimal hyperparameters? 

By the way, I'm working on refactoring the experiment code, especially sepsis data. I will let you know when it is ready.

**Comment by chenxiaodanhit** [comment]
Thank you for your patience! I encounter the same issue with the other models as well.

**Comment by yongkyung-oh** [comment]
I re-ran the code in a different environment and obtained similar results as follows:

**Neural LSDE**
![neurallsde](https://github.com/user-attachments/assets/3b35dc56-e48c-4097-8ab5-c3600fdcda79)
**Neural LNSDE**
![neurallnsde](https://github.com/user-attachments/assets/c2863c29-ebe1-478d-9755-b64d540f60d9)
**Neural GSDE**
![neuragsde](https://github.com/user-attachments/assets/bba86459-0fa6-4274-8b20-788a467a8e7e)

I'm not certain about the exact cause of the performance degradation, but it might be due to training instability. Therefore, I changed the scheduler and monitoring metric. Please check the updated code and refer to this [repo](https://github.com/sheoyon-jhin/ANCDE/blob/main/experiments/common.py) for a similar approach.

**Comment by chenxiaodanhit** [comment]
Thank you very much for your patience and response!
